02-peer-protocol.md

```diff
@@ -138,9 +177,9 @@ The sending node:
   - MUST set `first_per_commitment_point` to the per-commitment point to be used for the initial commitment transaction, derived as specified in [BOLT #3](03-transactions.md#per-commitment-secret-requirements).
   - MUST set undefined bits in `channel_flags` to 0.
   - if both nodes advertised the `option_upfront_shutdown_script` feature:
-	- MUST include either a valid `shutdown_scriptpubkey` as required by `shutdown` `scriptpubkey`, or a zero-length `shutdown_scriptpubkey`.
+  - MUST include either a valid `shutdown_scriptpubkey` as required by `shutdown` `scriptpubkey`, or a zero-length `shutdown_scriptpubkey`.
```

Comment:
original indentation was correct, as this is a sub requirement of the above 'if' condition. Though it should be indented two spaces in from parent

Reply:
Sorry; must be another case where my merging text caused spacing problems.

Reply:
Yes, fixed.